bin/shuji.js

```diff
@@ -157,33 +163,41 @@ if (!fs.existsSync(outputDir)) {
 }
 
 // Process then...
-fileList.forEach((filepath) => {
+fileList.forEach(async (inputFilepath) => {
   if (opts.verbose) {
-    console.log(`Processing file ${filepath}`);
+    console.log(`Processing file ${inputFilepath}`);
   }
 
-  const input = fs.readFileSync(filepath, 'utf8'),
-    outdir = path.join(outputDir, path.dirname(filepath)),
-    output = shuji(input, {
-      verbose: typeof opts.verbose === 'boolean' ?
-        opts.verbose :
-        false
-    });
+  const outdir = path.join(outputDir, path.dirname(inputFilepath));
+
+  const input = fs.readFileSync(inputFilepath, 'utf8');
+  console.log(input);
+  const output = await shuji(input, {
+    verbose: typeof opts.verbose === 'boolean' ?
+      opts.verbose :
+      false
+  });
 
   fs.ensureDirSync(outdir);
 
-  Object.keys(output).forEach((item) => {
-    const outfile = path.join(outdir, item);
+  const sourceFiles = Object.entries(output);
+
+  if (!sourceFiles.length) {
+    console.error(`Error: Could not reverse sourcemap for file ${inputFilepath}`);
+  }
+
+  sourceFiles.forEach(([filename, content]) => {
+    const outputFilepath = path.join(outdir, filename);
 
     if (opts.verbose) {
-      console.log(`Writing to file ${outfile}`);
+      console.log(`Writing to file ${outputFilepath}`);
     }
 
-    if (fs.existsSync(outfile)) {
-      console.error('File existed, skipping!');
+    if (fs.existsSync(outputFilepath)) {
+      console.error('Error: File already exists, skipping!');
```

Comment:
I don't see the need to write `Error` when already using `console.error`

Reply:
because all `console.error` does is write to stderr. (atleast for me on windows with git bash) it is not immediately obvious that a line of output came from stderr instead of stdout. Alternatively you could use something like chalk to color the output red to make it more clear that an error occurred and the user needs to try something different.

Reply:
for this specific console.error it might not really be necessary as it is not that critical of an error (so maybe it should say `Warning:`)